server/liveblog/embed/embed_assets/default-theme/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "default-theme",
+    "version": "1.0",
+    "angularModule": "liveblog.default-theme",
+    "styles": [
+        "styles/embed.css",
+        "//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css"
+    ],
+    "scripts": [
+        "//code.angularjs.org/1.3.14/angular-sanitize.js",
+        "//code.angularjs.org/1.3.14/angular-animate.js",
+        "main.js"
```

Comment:
Angular is already imported with the `liveblog-embed` module.
> main.js the outside of the theme and from the scope of the theme, so it shouldn't be in the theme

I don't understand sorry.
You prefer to have a `main.js` main with a forced name witch is always imported between two `before_script` and `after_script` templates instead of an explicit list of scripts ?